src/main/kotlin/com/cognifide/gradle/aem/api/AemConfig.kt

```diff
@@ -79,7 +79,7 @@ class AemConfig(
      * Defines maximum time after which initializing connection to AEM will be aborted (e.g on upload, install).
      */
     @Input
-    var instanceConnectionTimeout: Int = props.int("aem.instance.connectionTimeout", 5000)
```

Comment:
why?

Reply:
Made it a little bit more reasonable. 5s timeout caused build package operation to fail because of read timeout.